src/components/loader/loader-square.style.ts

```diff
@@ -52,10 +55,16 @@ const getDimentions = (
   `;
 };
 
+// TO-DO: Replace hard coded hex values with tokens if added in the future.
```

Comment:
comment: I think it would be better to raise tickets for these rather than add TODOs but I also don't mind if you leave them but include the ticket number etc